app/src/main/resources/log4j.properties

```
@@ -1,6 +1,6 @@
 log4j.rootCategory=INFO, CONSOLE
 
-LOG_PATTERN=[%d{yyyy-MM-dd HH:mm:ss.SSSXXX}] [%p] [%x] [%t] [%c] --- %m%n
+LOG_PATTERN=[%d{yyyy-MM-dd HH:mm:ss.SSSXXX}] [%p] [%x] [%t] [%c] --- %m %throwable{compact} %n
```

Not sure about the extra space here:
```suggestion
LOG_PATTERN=[%d{yyyy-MM-dd HH:mm:ss.SSSXXX}] [%p] [%x] [%t] [%c] --- %m %throwable{compact}%n
```

---

@a1exsh apologies, did not notice the suggestion

---

No worries :)